### DISCUSSION TITLE
Disable 'InstallButton' wont work

### DISCUSSION THREAD

**Discussion by Frittions** [wixtoolset/issues#7953] [Questions] [answered]
I want the user to review a 'Terms of Service' document and then click a checkbox before running the burn bootstrapper. Therefore, I would like the 'Install' button to be disabled (greyed out) until the user clicks the checkbox. However, I cannot seem to disable (either by EnableCondition or VisibleCondition) the 'Install' button. It looks to me that the 'Install' button is hard-coded to not be able to be disabled.

`<Button Name="InstallButton" X="-171" Y="-11" Width="75" Height="23" TabStop="yes" FontId="0" EnableCondition='PleaseConfirmCheckbox="1"' >
	<Text>#(loc.InstallButton)</Text>
</Button>`

Is there really no way to disable the 'Install' button in the burn bootstrapper? I am using the  'WixStandardBootstrapperApplication' bootstrapper application if that matters.

**Comment by robmen** [comment, marked as answer]
I'm pretty sure I've seen this done before.

**Comment by ackh** [comment]
@Frittions I'm facing a similar problem and getting to know your solution might help me and others in the future. Would you mind sharing it?

**Reply by Frittions** [reply]
I was never able to find a solution. I eventually gave up and worked around the issue.

**Reply by ackh** [reply]
Thanks for taking the time to reply nonetheless! I thought I'm missing something because apparently someone else could make it work.

I had a look at the code of the default bootstrapper application and from what I see there, whether the "Install" button is enabled or disabled in hard-wired in the bootstrapper itself (see [here](https://github.com/wixtoolset/wix/blob/64fc1bc29460db8b525d2817078e1a40a197b4c9/src/ext/Bal/stdbas/WixStandardBootstrapperApplication.cpp#L3900)). For my case, I was attempting to have a second checkbox but given that the logic is in the bootstrapper and not on the XML level would mean I have to customize the bootstrapper.
